packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -0,0 +1,348 @@
+import React, { useEffect, useState } from "react";
+import { RiVideoUploadLine } from "react-icons/ri";
+import { Dayjs } from "dayjs";
+
+import {
+    FluidLayout,
+    isValidDelegate,
+    onError,
+    uploadIpfsFileWithTransaction,
+    uploadToIpfs,
+    useCommunityGlobals,
+    useCurrentElection,
+    useLocalStorage,
+    useOngoingElectionData,
+    useUALAccount,
+} from "_app";
+import {
+    Container,
+    Text,
+    Expander,
+    Heading,
+    Loader,
+    VideoSubmissionFormAndPreview,
+    VideoSubmissionPhase,
+    Button,
+    Link,
+} from "_app/ui";
+import {
+    CurrentElection,
+    ElectionStatus,
+    ErrorLoadingElection,
+    setElectionRoundVideo,
+    getRoundTimes,
+} from "elections";
+import { EncryptionPasswordAlert } from "encryption";
+import { EdenMember } from "members";
+import { RoundHeader } from "elections/components/ongoing-election-components";
+
+// TODO:
+// 1) add buttons that launch video upload service in new tab
+// 2) test inductions to ensure I haven't screwed up their video upload
+// 3) Test video uploads during (and after) an election
```

Comment:
question: Where are we with this? Do we have the buttons yet?

Reply:
nice catch, we have the buttons already, I will remove these todo items from the code because we are already on it.